envreader/bool.go

```diff
@@ -0,0 +1,18 @@
+package envreader
+
+import (
+	"fmt"
+	"strconv"
+)
+
+func (r *EnvReader) BoolOrDefault(key string, def bool) bool {
+	var ret bool
+	val := r.Env(key)
+	fmt.Println(val)
+	if val == "" {
+		return def
+	}
+	
+	ret, _ = strconv.ParseBool(val)
```

Comment:
A validação do erro aqui é importante. ParseBool não deve aceitar qualquer string. Ele deve ser capaz de parsear "true" ou "false".
Por ex: e se eu usar
MYVAR=feijoada
r.BoolOrDefault("MYVAR", true)? 
```suggestion
        if ret, err := strconv.ParseBool(val); err == nil {
                return val
        }
```

Reply:
faz total sentido!